node/src/components/event_stream_server.rs

```diff
@@ -234,64 +295,7 @@ where
         &mut self,
         _effect_builder: EffectBuilder<REv>,
     ) -> Result<Effects<Self::ComponentEvent>, Self::Error> {
-        let cfg = self.config.clone();
```

Comment:
just cuz?

Reply:
I moved the call to `clone()` to the last consumer in https://github.com/casper-network/casper-node/pull/3396/commits/12a53e511c8a0dc8befb3538bb1a1243b25b5717